### DISCUSSION TITLE
GitHub Overview - 2020-01-27

### DISCUSSION THREAD

**Discussion by jwildfire** [SafetyGraphics/safetyGraphics#485] [Meetings]
# Attendees
- @jwildfire 
- Lots of others 😬 

# Agenda/Notes

1. Markdown Overview - https://guides.github.com/features/mastering-markdown/
2. Github Project Management Tools
   - Discussions
   - Issues
   - Websites
   - Projects
   - Wikis
3. Technical project management
  - Intro docs
      - https://github.com/SafetyGraphics/SafetyGraphics.github.io/blob/master/CONTRIBUTING.md
      - https://github.com/RhoInc/open-source-handbook
  - Full Tutorial coming after v2!
4. Q&A

**Comment by tbajn** [comment]
Thanks, Jeremy, for today's tutorial. Does this count as completing my assignment?

**Comment by wal615** [comment]
@jwildfire, Thanks for the tutorial, which is very helpful!
